linter_plugin.py

```diff
@@ -1,31 +1,52 @@
-"""Certbot ACME PyLint plugin.
-
+"""Certbot PyLint plugin.
 http://docs.pylint.org/plugins.html
-
 """
-from astroid import MANAGER
-from astroid import nodes
-
-
-def register(unused_linter):
-    """Register this module as PyLint plugin."""
-    pass
-
-
-def _transform(cls):
-    # fix the "no-member" error on instances of
-    # letsencrypt.acme.util.ImmutableMap subclasses (instance
-    # attributes are initialized dynamically based on __slots__)
-
-    # TODO: this is too broad and applies to any tested class...
-
-    #if cls.slots() is not None:
-    #    for slot in cls.slots():
-    #        cls.locals[slot.value] = [nodes.EmptyNode()]
-
-    if cls.name == 'JSONObjectWithFields':
-        # _fields is magically introduced by JSONObjectWithFieldsMeta
-        cls.locals['_fields'] = [nodes.EmptyNode()]
-
-
-MANAGER.register_transform(nodes.Class, _transform)
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers.utils import check_messages
+
+
+# Module in theses packages can import os module.
+WHITELIST_PACKAGES = ['acme', 'certbot_compatibility_test', 'letshelp_certbot', 'lock_test']
+
+
+class ForbidStandardOsModule(BaseChecker):
+    """
+    This checker ensures that standard os module is not imported by certbot classes.
```

Comment:
I'd either delete the word classes or change it to modules.

Reply:
Done (/classes/modules/).